moped-editor/src/views/projects/projectsListView/ProjectsListViewTable.js

```diff
@@ -298,8 +298,50 @@ const ProjectsListViewTable = ({ title, query, searchTerm, referenceData }) => {
       field: "updated_at",
       render: entry => new Date(entry.updated_at).toLocaleDateString("en-US"),
     },
+    {
+      title: "Task order",
+      field: "task_order",
+      hidden: true,
+      emptyValue: "-",
+      render: entry => {
+        // Empty value won't work in some cases where task_order is an empty array.
+        if (entry?.task_order.length < 1) {
+          return "-";
+        }
+        // Render values as a comma seperated string
+        let content = entry.task_order
+          .map(taskOrder => {
+            return taskOrder.display_name;
+          })
+          .join(", ");
+
+        return <div style={{ maxWidth: "265px" }}>{content}</div>;
+      },
+    },
+    {
+      title: "Contractor/Contract",
+      field: "contractor",
+      hidden: true,
+      emptyValue: "-",
+      render: entry => (entry.contractor === "" ? "-" : entry.contractor),
```

Comment:
The render line seems to re-implement the expected behavior from the `emptyValue` configuration. Not a problem, just seemed to stick out.

Reply:
this is a quirk of material table, if it is null or undefined then it will render the emptyValue "-"
But if it is an empty string, material table doesn't identify that as empty so it would render the empty string. This way it looks consistent. There isn't a way for someone to delete a contractor besides saving an empty string.